addons/mail/tests/common.py

```diff
@@ -663,23 +690,29 @@ def _assertMailMail(self, mail, recipients_list, status,
                         recipient,
                         **(email_values or {})
                     )
+            if email_to_all:
+                self.assertSentEmail(
+                    email_values['email_from'] if email_values and email_values.get('email_from') else author,
+                    email_to_all,
+                    **(email_values or {}))
 
     def assertMailMail(self, recipients, status,
                        email_to_recipients=None,
-                       mail_message=None, author=None,
+                       mail_message=None, author=None, email_to_all=None,
                        content=None, fields_values=None, email_values=None):
         """ Assert mail.mail records are created and maybe sent as emails. This
         method takes partners as source to find mails and check their content.
         See '_assertMailMail' for more details.
 
         :param recipients: a ``res.partner`` recordset. See
           ``_find_mail_mail_wpartners``;
+        :param email_to_all: list of email addresses. See ``_find_mail_mail_wemails``;
         :param mail_message: used to find the related email;
 
         See '_assertMailMail' for more details about other parameters.
         """
-        found_mail = self._find_mail_mail_wpartners(
-            recipients, status, mail_message=mail_message,
+        found_mail = self._find_mail_mail_wpartners_or_emails(
```

Comment:
Would probably call _find_mail_mail_wpartners or _find_mail_mail_wemails depending on given input. Would be clearer to me :) .